src/extras.py

```diff
@@ -1,27 +1,37 @@
 from decimal import Decimal
-from typing import Dict, Callable
+from functools import lru_cache
+from typing import Dict, Callable, Optional
 
 from flask import render_template
 from soda_api import client, SALARY_DATASET
 from api_types import Record, DEFAULT_DATASET
 
 
-def _augment_with_salary(record: Record) -> str:
-    last = record["last_name"]
-    first = record["first_name"]
+@lru_cache(maxsize=1000)
```

Comment:
Glad we have this, it's slow as hell otherwise.

Reply:
Lol and it's a one line change too! After moving it out into something that's cached easily